docs/synapse/userguides/storm_ref_data_mod.ipynb

```diff
@@ -502,6 +504,204 @@
+   "source": [
+    "# Make some nodes\n",
+    "q = '[ inet:fqdn=newsonet.net inet:fqdn=staycools.net +#aka.feye.thr.apt1 ]'\n",
+    "# Make some light edges\n",
+    "q1 = \"media:news=a3759709982377809f28fc0555a38193 [ +(refs)> { +#aka.feye.thr.apt1 } ]\"\n",
+    "# Display the syntax\n",
+    "print(q1)\n",
+    "# Run the queries and test\n",
+    "podes = await core.eval(q, num=2, cmdr=False)\n",
+    "podes = await core.eval(q1, num=1, cmdr=False)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "*Link the specified inet:cidr4 netblock to any IP address within that netblock that exists in the Cortex ( as referenced by the Storm expression ) using a \"hasip\" light edge:*"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": true
+   },
+   "outputs": [],
+   "source": [
+    "# Make some nodes\n",
+    "q = '[ inet:cidr4=123.120.96.0/24 inet:ipv4=123.120.96.12 inet:ipv4=123.120.96.53 inet:ipv4=123.120.96.202 ]'\n",
+    "# Make some light edges\n",
+    "q1 = \"inet:cidr4=123.120.96.0/24 [ +(hasip)> { inet:ipv4=123.120.96.0/24 } ]\"\n",
+    "# Display the syntax\n",
+    "print(q1)\n",
+    "# Run the queries and test\n",
+    "podes = await core.eval(q, num=4, cmdr=False)\n",
+    "podes = await core.eval(q1, num=1, cmdr=False)"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "**Usage Notes:**\n",
+    "\n",
+    "- No light edge verbs exist in a Cortex by default; they must be created.\n",
+    "- Light edge verbs are created at the user's discretion \"on the fly\" (i.e., when they are first used to link nodes); they do not need to be created manually before they can be used.\n",
+    "\n",
+    "  - We recommend that users agree on a consistent set of light edge verbs and their meanings.\n",
+    "  - The Storm :ref:`storm-model` commands can be used to list and work with any light edge verbs in a Cortex.\n",
+    "\n",
+    "- A light edge's verb typically has a logical direction (a report \"references\" a set of indicators that it contains, but the indicators do not \"reference\" the report). However, it is up to the user to create the light edges in the correct direction and using forms that are sensical for the light edge verb. That is, there is nothing in the Storm syntax iitself to prevent users linking any arbitrary nodes in arbitrary directions using arbitrary light edge verbs.\n",
+    "- The plus sign ( ``+`` ) used with the light edge expression within the edit brackets is used to create the light edge(s).\n",
+    "- Light edges can be created in either \"direction\" (e.g., with the directional arrow pointing either right ( ``+(<verb>)>`` ) or left ( ``<(<verb>)+`` ) - whichever syntax is easier.\n",
+    "- The nodes referenced by the Storm expression (i.e., within the curly braces) are **NOT** created by the syntax above. They must already exist in order for the light edges to be created."
```

Comment:
the storm expression is arbitrary and can be used to make nodes or do anything else. the nodes emitted are the ones linked.